### ISSUE TITLE
Download links should lead to standalone DEX apps

### ISSUE THREAD

**Issue by norwnd** [decred/dexweb#41]
After DEX standalone apps are available fully (eg. https://github.com/decred/dcrdex/pull/2333, https://github.com/decred/dcrdex/pull/2351) it makes sense to distribute those instead of Decrediton (like it currently does on [this page](https://dex.decred.org/#downloads)):

<img width="500" alt="image" src="https://github.com/decred/dexweb/assets/112318969/8502a846-e538-49e7-8b5d-c7ac4a367ac1">

**Comment by norwnd** [comment]
Perhaps this footer also needs adjustments, once standalone apps are finished (it currently links to Github releases page):

<img width="400" alt="image" src="https://github.com/decred/dexweb/assets/112318969/313bfb21-487d-44ed-8aae-ca42af0187cf">

**Comment by xaur** [comment]
Agreed. But don't fully remove Decrediton (it still may be a better option for some users), just make the standalone app a more visible and recommended option.

Currently it is reversed: first you get the 3 big buttons to donwload Decrediton, then 1 big button for Umbrel, than the "alternative" download link "Click here to download the standalone DCRDEX application instead".

**Comment by xaur** [comment]
Thinking more about onboarding flows (https://github.com/decred/dcrdex/issues/2563) I think the issue here is "choices are hard" and the fix is to guide through the choices and explain them.

Issues I see with the downloads block:

- "Standalone" and "Decrediton" are not explained for newcomers, it is hard to make a choice
- Umbrel block looks like a Decrediton option but it is a standalone option
- Standalone is not the default option as the first comment says

To address these the [downloads.html](https://github.com/decred/dexweb/blob/master/src/layouts/partials/downloads.html) partial can be restructured as follows:

- Show two big logical sections for the standalone app and Decrediton (separate them visually with boxes or something)

- Standalone app section
  - Short paragraph explaining the benefits
  - A few screenshot thumbnails (#20)
  - Install links: Windows, macOS, Linux, (other OSes), Umbrel, AUR, [other OS packages](https://github.com/decred/dcrdex#os-packages) when they are added

- Decrediton section
  - Short paragraph explaining the benefits
  - A few screenshot thumbnails
  - Install links: Windows, macOS, Linux

The other partials (`intro` and `get-started`) are good, except I'd tweak the `1. Intro` to point to the revised download sections instead of directly to GitHub.
